scripts/tests/traits/dual_wield.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+-- Dual Wield Trait - Equipment TP Reset Prevention
+-- This test verifies that TP is preserved when equipment changes
+-- fail due to missing Dual Wield trait.
+--
+-- Note: Dual Wield is available on:
+--   - NIN (Ninja) - Level 10
+--   - DNC (Dancer) - Level 20
+--
+-- Fixes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/2268
+-----------------------------------
+
+describe('Dual Wield Trait - Equipment TP Reset Prevention', function()
+    ---@type CClientEntityPair
+    local player
+
+    before_each(function()
+        -- Spawn NIN/5 (before DW unlocks at level 10)
+        player = xi.test.world:spawnPlayer({
+            level = 5,
+            mjob = xi.job.NIN,
+            zone = xi.zone.WEST_RONFAURE
+        })
+    end)
+
+    it('should preserve TP when offhand equip fails without Dual Wield trait', function()
+        -- Arrange: Give player weapons and TP
```

Comment:
I mean, sure, Arrange/Act/Assert is right out of a textbook, but `describe(...)`, `it(...)`, and the inline comment in `assert(...)` are plenty to give context to what's going on here. Lots of these comments aren't adding any value

Reply:
Would you want me to remove Arrange / Act / Assert delineations completely? I have removed a lot of the unnecessary fluff.